### ISSUE TITLE
Support custom themes

### ISSUE THREAD

**Issue by simc** [invertase/zapp.run#54]
Having at least some level of customization for example editor background color would be really cool! Especially for embedded editors.

A VSCode-like json file in the root of a repository would be ideal imo

**Comment by Ehesp** [comment]
We actually had a proof of concept for choosing loads of different themes - think it's still on a branch somewhere. The problem we had was that the editor looked a bit odd if it was a random theme within the rest of the app. So we synced the entire application to the theme, but you start to have issues with some next not being visible and stuff.

I'll try and find some screenshots.

Worth noting too, that VSCode themes don't directly map to Monaco (the editor we use), so it's not so simple to just support any theme.

**Comment by leisim** [comment]
I understand what you are saying. My thinking was that for embedded editors it might look weird if the host background is slightly different 🤷🏼‍♂️

**Comment by Ehesp** [comment]
Yeah maybe themes for embedding only makes sense. I'll have a play around.

**Comment by Solido** [comment]
Follow-up on that point too. 

Zapp take another dimension when embedded and I'm also looking for a visual integration.

Not sure if the theme should be linked to a project or an account.
I would rather upload them to my account then having a default one and being able to override the selected theme using the url else and update to the theme would force update every single project.

Thank you!
